homeassistant/components/android_ip_webcam/entity.py

```diff
@@ -19,11 +19,6 @@ def __init__(
     ) -> None:
         """Initialize the base entity."""
         super().__init__(coordinator)
-        if CONF_NAME in coordinator.config_entry.data:
```

Comment:
We normally mark configuration changes as Deprecated, and wait 5 months before removing, I cannot see this here ??

Reply:
This is not a YAML related change (to which that ADR applies).